content/2015.07.cliquet.rationale-en.rst

```diff
@@ -0,0 +1,494 @@
+Why did we make Cliquet?
```

Comment:
Also note that the french verbe ``faire`` is often use for diverse things other than ``make``. IMHO, if you use build instead it should do the trick.